src/hotspot/share/utilities/macros.hpp

```diff
@@ -537,6 +537,14 @@
 #define NOT_AARCH64(code) code
 #endif
 
+#ifdef TARGET_ARCH_aarch64
+#define AARCH64_PORT_ONLY(code) code
+#define NOT_AARCH64_PORT_ONLY(code)
+#else
```

Comment:
I don't think we need `NOT_AARCH64_PORT_ONLY`, and it's too confusing. Otherwise OK.

Reply:
Agreed. Only added it to keep with the style of the file. Will remove.